Client/CMakeLists.txt

```diff
@@ -5,12 +5,12 @@ project(R_Type_Client)
 add_subdirectory(UnitTests)
 add_subdirectory(SFML EXCLUDE_FROM_ALL)
 include_directories(SYSTEM "SFML/include/")
-
+include_directories(../Shared/LibraryLoader/ ../Shared/Libs/Interfaces ../Shared/Include)
 
 IF (MSVC)
-    set(CMAKE_CXX_FLAGS "-std=c++11")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 ELSE()
-    set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -Wextra -pedantic -Werror -Wuninitialized -Winit-self -Weffc++ -Wno-unused-parameter")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -Werror -Wuninitialized -Winit-self -Weffc++ -Wno-unused-parameter")
 ENDIF()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Flags are essential to avoid runtime inconsistencies between OS. -Weffc++ is the most important one.
We don't set them on Windows because they don't exist there ^^ CMake uses MSVC on Windows which doesn't implement GCC's flags.